src/androidTest/java/com/mixpanel/android/mpmetrics/TestUtils.java

```diff
@@ -30,7 +30,15 @@ public CleanMixpanelAPI(final Context context, final Future<SharedPreferences> r
         /* package */ PersistentIdentity getPersistentIdentity(final Context context, final Future<SharedPreferences> referrerPreferences, final String token) {
             final String prefsName = "com.mixpanel.android.mpmetrics.MixpanelAPI_" + token;
             final SharedPreferences ret = context.getSharedPreferences(prefsName, Context.MODE_PRIVATE);
-            ret.edit().clear().putBoolean("tracked_integration", true).commit();
+            ret.edit().clear().commit();
+
+            final String timeEventsPrefsName = "com.mixpanel.android.mpmetrics.MixpanelAPI.TimeEvents_" + token;
+            final SharedPreferences timeSharedPrefs = context.getSharedPreferences(timeEventsPrefsName, Context.MODE_PRIVATE);
+            timeSharedPrefs.edit().clear().commit();
+
+            final String mixpanelPrefsName = "com.mixpanel.android.mpmetrics.Mixpanel";
+            final SharedPreferences mpSharedPrefs = context.getSharedPreferences(mixpanelPrefsName, Context.MODE_PRIVATE);
+            mpSharedPrefs.edit().clear().putBoolean(token, true).putBoolean("has_launched", true).commit();
 
             return super.getPersistentIdentity(context, referrerPreferences, token);
```

Comment:
i couldn't find any specific unit tests for automaticEvents, do you feel good about the overall coverage?